src/TensorFlowNET.Core/Keras/Activations/Activations.cs

```diff
@@ -25,10 +26,52 @@ public class Activation
             };
         }
     }
+    public class ActivationAdaptor
```

Comment:
Providing some constructors and adding descriptions for usage is better, because one of the properties must be specified when using it.

Reply:
Thank you for your feedback. I will make the suggested improvements and resubmit the pull request for your review.

Reply:
After opening a pull request, the commits after that will update the status of PR. Therefore when in review, just making an extra commit is ok. :)

Reply:
Got it!✔